packages/web-components/src/components/cbp-notice/cbp-notice.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * @prop --cbp-list-padding: var(--cbp-space-4x);
+*/
+
+:root {  
```

Comment:
Your naming isn't consistent with other components. Please do --cbp-notice-color -bg -border, etc. You don't need to use "left" the border color.

Reply:
updated